docs/source/kubernetes/ovh/step-zero-ovh.md

```diff
@@ -1,15 +1,15 @@
 (ovh)=
 
-# Kubernetes on [OVHcloud](https://www.ovh.ie/) (OVH)
+# Kubernetes on [OVHcloud](https://www.ovhcloud.com/en-ie/) (OVH)
```

Comment:
```suggestion
# Kubernetes on [OVHcloud](https://www.ovhcloud.com/) (OVH)
```
I don't think there's anything region specific, so using the top-level URL should automatically redirect to the user's local region (or something like that, e.g. I'm automatically taken to https://www.ovhcloud.com/en-gb/)